nginx/nginx.conf

```diff
@@ -0,0 +1,21 @@
+events { }
+http {
+  include mime.types;
+  server {
+    server_name docs.stellar.org;
```

Comment:
It would be good if the nginx accepted any server_name. Reason for that is that we could then use the same image for any dev or PR-preview style deployment without need to modify it.

Reply:
I copied the format from [stellar.org-config](https://github.com/stellar/stellar.org-config/blob/master/nginx/nginx.conf), will need some guidance on how to change it if this doesn't work.

Reply:
Ah interesting. So it works because the first vhost is also the default one ([docs](https://nginx.org/en/docs/http/request_processing.html)).
So it's fine to merge as is but if you want to be explicit you could do something like this:
```
server {
    listen      80 default_server;
    server_name docs.stellar.org;
}
```